src/masternodeman.h

```diff
@@ -26,7 +26,7 @@ class CMasternodeMan
 
     static const int DSEG_UPDATE_SECONDS        = 3 * 60 * 60;
 
-    static const int LAST_PAID_SCAN_BLOCKS      = 100;
+    static const int LAST_PAID_SCAN_BLOCKS;
```

Comment:
Why is the value (`= 100`) moved to the cpp file while all other consts are declared here?

Reply:
Because otherwise it fails to link :/

```
Undefined symbols for architecture x86_64:
  "CMasternodeMan::LAST_PAID_SCAN_BLOCKS", referenced from:
      CMasternodeMan::UpdateLastPaid(CBlockIndex const*) in libbitcoin_server.a(libbitcoin_server_a-masternodeman.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Can't reproduce it on GCC,  looks like clang feature.) I still can't understand why the same error does not appears for other constants like DSEG_UPDATE_SECONDS,  that is defined in *.h and is used in *.cpp like LAST_PAID_SCAN_BLOCKS is,  but it is not really important. utACK.

Reply:
Can't reproduce it on GCC, looks like clang feature.) I still can't understand why the same error does not appears for other constants like DSEG_UPDATE_SECONDS, that is defined in *.h and is used in *.cpp like LAST_PAID_SCAN_BLOCKS is, but it is not really important. utACK.